projects/goharbor/harbor-notary/GIT_TAG

```diff
@@ -0,0 +1 @@
+v0.7.0
```

Comment:
Harbor uses notary v0.6.1 that doesn't have go.mod file, which is required in our build-tooling. So I bump that up but need to be tested.